README.md

```diff
@@ -92,6 +94,35 @@ async function subsystem (fastify, opts) {
 }
 ```
 
+If you want to change the Fastify hook that the middleware will be attached to, pass a `hook` option like so:
+
+```js
+const fastify = require('fastify')()
+
+fastify
+  .register(require('middie'), { hook: 'preHandler' })
+  .register(subsystem)
+
+async function subsystem (fastify, opts) {
```

Comment:
This is a regular Fastify plugin issuing things during the regular life cycle. Given that this example is about `middie`, there should be something that shows a "middleware" now executing during Fastify's `preHandler` phase.

Reply:
As you can see in the example, the plugin call 'use', i.e. middie.